CMakeLists.txt

```diff
@@ -43,28 +47,28 @@ INCLUDE(EnableWarnings)
 # Build options
 #
 OPTION(SONAME				"Set the (SO)VERSION of the target"			 ON)
-OPTION(BUILD_SHARED_LIBS		"Build Shared Library (OFF for Static)"			 ON)
+OPTION(BUILD_SHARED_LIBS	"Build Shared Library (OFF for Static)"		 ON)
 OPTION(THREADSAFE			"Build libgit2 as threadsafe"				 ON)
 OPTION(BUILD_CLAR			"Build Tests using the Clar suite"			 ON)
-OPTION(BUILD_EXAMPLES			"Build library usage example apps"			OFF)
-OPTION(BUILD_FUZZERS			"Build the fuzz targets"				OFF)
-OPTION(TAGS				"Generate tags"						OFF)
+OPTION(BUILD_EXAMPLES		"Build library usage example apps"			OFF)
+OPTION(BUILD_FUZZERS		"Build the fuzz targets"					OFF)
+OPTION(TAGS					"Generate tags"								OFF)
 OPTION(PROFILE				"Generate profiling information"			OFF)
-OPTION(ENABLE_TRACE			"Enables tracing support"				OFF)
-OPTION(LIBGIT2_FILENAME			"Name of the produced binary"				OFF)
+OPTION(ENABLE_TRACE			"Enables tracing support"					OFF)
+OPTION(LIBGIT2_FILENAME		"Name of the produced binary"				OFF)
 
-   SET(SHA1_BACKEND 			"CollisionDetection"			       CACHE STRING
+   SET(SHA1_BACKEND 			"CollisionDetection" CACHE STRING
        "Backend to use for SHA1. One of Generic, OpenSSL, Win32, CommonCrypto, mbedTLS, CollisionDetection.")
-OPTION(USE_SSH				"Link with libssh2 to enable SSH support"		 ON)
-OPTION(USE_HTTPS			"Enable HTTPS support. Can be set to a specific backend" ON)
-OPTION(USE_GSSAPI			"Link with libgssapi for SPNEGO auth"			OFF)
-OPTION(USE_STANDALONE_FUZZERS		"Enable standalone fuzzers (compatible with gcc)"	OFF)
-OPTION(VALGRIND				"Configure build for valgrind"				OFF)
-OPTION(USE_EXT_HTTP_PARSER		"Use system HTTP_Parser if available"			 ON)
-OPTION(DEBUG_POOL			"Enable debug pool allocator"				OFF)
-OPTION(ENABLE_WERROR			"Enable compilation with -Werror"			OFF)
-OPTION(USE_BUNDLED_ZLIB    		"Use the bundled version of zlib"			OFF)
-OPTION(DEPRECATE_HARD			"Do not include deprecated functions in the library"	OFF)
+OPTION(USE_SSH					"Link with libssh2 to enable SSH support"		 			ON)
+OPTION(USE_HTTPS				"Enable HTTPS support. Can be set to a specific backend"	ON)
+OPTION(USE_GSSAPI				"Link with libgssapi for SPNEGO auth"						OFF)
+OPTION(USE_STANDALONE_FUZZERS	"Enable standalone fuzzers (compatible with gcc)"			OFF)
+OPTION(VALGRIND					"Configure build for valgrind"								OFF)
+OPTION(USE_EXT_HTTP_PARSER		"Use system HTTP_Parser if available"						ON)
+OPTION(DEBUG_POOL				"Enable debug pool allocator"								OFF)
+OPTION(ENABLE_WERROR			"Enable compilation with -Werror"							OFF)
+OPTION(USE_BUNDLED_ZLIB			"Use the bundled version of zlib"							OFF)
+OPTION(DEPRECATE_HARD			"Do not include deprecated functions in the library"		OFF)
```

Comment:
Please avoid such major reformatting, as it makes it quite hard to see what was actually changed

Reply:
VS Code shows these lines as not whitespace aligned, while GH shows them aligned. I don't know why. I'll revert.